src/Config.php

```diff
@@ -28,50 +28,57 @@ class Config
 {
     /**
      * Input from user. This is either a file path, a string or an array
+     *
      * @var string|array
      */
     protected $input = null;
 
     /**
      * Array of logger configuration options: (logger attributes, formatters, handlers, etc.)
+     *
      * @var array
      */
     protected $options = array();
 
     /**
      * Array of Formatter objects
+     *
      * @var Monolog\Formatter\FormatterInterface[]
      */
     protected $formatters = array();
 
     /**
      * Array of Handler objects
+     *
      * @var Monolog\Handler\HandlerInterface[]
      */
     protected $handlers = array();
 
     /**
      * Array of Processor objects
+     *
      * @var callable[]
      */
     protected $processors = array();
 
     /**
      * Array of logger objects
+     *
      * @var Monolog\Logger[]
      */
     protected $loggers = array();
 
     /**
      * Config loader
+     *
      * @var ConfigLoader
      */
     protected $loader = null;
 
     /**
      * Instantiate a Config object
      *
-     * @param string|array $input User input
+     * @param string|array $input  User input
```

Comment:
Yeah, according to `phpcs`, which Travis uses. It aligns the parameter name with the one just below it.